kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/def/DataSetDefRegistry.java

```diff
@@ -45,6 +45,15 @@
      *         does not exist or there is no preprocessor registered.
      */
     List<DataSetPreprocessor> getDataSetDefPreProcessors(String uuid);
+
+    /**
+     * Get the post processors for a given definition for the specified data set.
+     *
+     * @param uuid The unique universal identifier of the data set
+     * @return The data set post processors list or null if the definition 
+     *         does not exist or there is no post processor is registered.
```

Comment:
remove duplicated is (is registered should be just registered)

Reply:
fixed